### DISCUSSION TITLE
What is the best way to check if a sprite is at the edge of a platform?

### DISCUSSION THREAD

**Discussion by acousticseagull** [replit/kaboom#212] [Q&A]
Any help would be much appreciated.

**Comment by slmjkdbtl** [comment]
There's `obj.grounded()` (from `body()`) that checks if is currently grounded or not, if you know the platform game obj you can also use `obj.isTouching(platform)` (from `area()`)

**Reply by acousticseagull** [reply]
Ok, thanks! I will give that a try.
